test/unit/test_diskfile.py

```diff
@@ -292,16 +306,33 @@ def test_put_meta_with_no_metadata(self):
         self.assertRaisesRegexp(SproxydHTTPException, 'no usermd',
                                 sfs.put_meta, 'object_name_1', None)
 
-    @mock.patch('swift.common.bufferedhttp.http_connect_raw',
-                return_value=FakeHTTPConn())
+    @mock.patch('urllib3.HTTPConnectionPool.request',
+                return_value=urllib3.response.HTTPResponse(status=200))
     def test_del_object(self, mock_http):
         sfs = SproxydFileSystem({}, mock.Mock())
         sfs.del_object('object_name_1')
 
-        self.assertEqual(1, mock_http.call_count)
-        mock_http.assert_called_with(mock.ANY, mock.ANY, 'DELETE',
-                                     sfs.base_path + 'object_name_1',
-                                     None)
+        mock_http.assert_called_once_with('DELETE',
+                                          sfs.base_path + 'object_name_1',
+                                          headers=None, preload_content=False)
+
+    def test_get_object(self):
+        server = eventlet.listen(('127.0.0.1', 0))
+        (ip, port) = server.getsockname()
+
+        def hello_world(env, start_response):
+            start_response('200 OK', [('Content-Type', 'text/plain')])
+            return ['Hello, World!\r\n']
```

Comment:
Use a variable containing this string, in the parent scope, to be reused below (see next comment).